src/styles/index.css

```diff
@@ -0,0 +1,128 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  padding: 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+}
+
+.card__image {
+  margin: 16px auto 0;
+  height: 134px;
+  background-image: url(../images/imac.jpeg);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: 160px 134px;
+}
+
+.card__name {
+  margin-top: 40px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  text-align: left;
+  color: #060b35;
+}
+
+.card__code {
+  margin-top: 4px;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  text-align: left;
+  color: #616070;
+}
+
+.card__price {
+  margin-top: 24px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.card__text {
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__cost {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__button {
+  height: 40px;
+}
+
+.card__buttonContent {
+  margin-top: 16px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
+  text-decoration: none;
+  border-radius: 5px;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  color: #fff;
+  background-color: #00acdc;
+  border: 1px solid #00acdc;
+}
+
+.card__buttonContent:hover {
+  color: #00acdc;
+  background-color: #fff;
+}
+
+.stars {
+  margin-top: 16px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars__stars {
+  display: flex;
+  gap: 4px;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__stars .stars__star:nth-child(1n) {
```

Comment:
The selector '.stars__stars .stars__star:nth-child(1n)' is equivalent to '.stars__stars .stars__star'. Consider simplifying the selector by removing ':nth-child(1n)'.